tox.ini

```diff
@@ -25,13 +25,15 @@ install_command =
 extras =
     test
     drf
-deps = django22: Django>=2.2,<2.3
        django32: Django<4.0
```

Comment:
This looks like a mistake, the `deps` keyword was removed, and I don't think tests ran with correct dependencies. Interesting that tox ran at all.